datajunction-server/datajunction_server/api/attributes.py

```diff
@@ -121,18 +132,25 @@ def default_attribute_types(session: Session = Depends(get_session)):
             set(default_attribute_type_names.keys()),
         ),
     )
-    attribute_types = session.exec(statement).all()
+    attribute_types = session.execute(statement).scalars().all()
     for type_ in attribute_types:
-        updated_type = default_attribute_type_names[type_.name].dict(
-            exclude_unset=True,
-        )
-        type_ = type_.update(updated_type)
-        session.add(type_)
+        if type_:  # pragma: no cover
```

Comment:
Why is this no cover needed if the code it replaced was covered?

Reply:
Oops, yeah, it is still covered and I just removed the `no cover` label.